README.md

```diff
@@ -95,6 +95,7 @@ Options
 - `g:ros_make` [current|all] Controls which package to build
 - `g:ros_build_system` [catkin|rosbuild|catkin-tools] Which build system to use
 - `g:ros_catkin_make_options` Additional options for catkin_make (i.e '-j4 -DCMAKE_BUILD_TYPE=Debug' ...)
+- `g:vimros_disable_python2_warning` Disable the warning about missing Python2 feature, defaults to 0.
```

Comment:
For consistency with all other options (there are more that are not listed here), please drop the `vim` prefix.

Reply:
I noticed that other variable were called `g:ros` but called it `g:vimros` to express the fact that this is not an option for ROS but for `vim-ros`. Can you confirm you really want `g:ros_`?

Reply:
Yeah I understand what you mean. But I don't think we can clearly separate what is an "option for ROS" and what is an "option for `vim-ros`". Take `g:ros_build_system`, for example. It's a ROS option since it corresponds to the build system used in ROS. But also it's a plugin option because it controls which executable the plugin invokes. I'd prefer to avoid situation where we need to apply fuzzy logic to decide what is `g:ros_` and what is `g:vimros_`.

Another motivation to have the same prefix is that it's a wide-spread convention in Vim world that the options provided by a plugin all have the same prefix (typically the plugin name).

Perhaps `g:vimros_` prefix could have been a better choice, but it's too late to change now.

Reply:
For sure, no problem, see next commits.